1.train_models/README.md

```diff
@@ -0,0 +1,10 @@
+# Develop the NF1 model
+In this module, we extract patterns to discern between cells with WT and Null genotypes using cell morphologies from plates 3, 3 prime, and 5.
+After substantial development, we have developed and optimized our final NF1 model with the same architecture as a logistic regression.
+The training and testing sets were choosen to create a 90-10 percent train-test split between wells, while approximately retaining a uniform genotype class distribution of cells within each data split.
```

Comment:
```suggestion
The training and testing sets were manually chosen to create a 90-10 percent train-test split between wells while approximately retaining a uniform genotype class distribution of cells within each data split.
```

Reply:
Is this accurate to use manual here? If not, recommend clarifying what was manually done.

Reply:
I think so. I just thought of a better way to phrase this, which will be applied shortly